specs/operator/manual-approval-gate.spec

```diff
@@ -0,0 +1,38 @@
+PIPELINES-28
+# ManualApprovalGate Pipelines operator specs
+
+## Approve Manual Approval gate pipeline: PIPELINES-28-TC01
+Tags: approvalgate, e2e, taskrun, sanity
+Component: Operator
+Level: Integration
+Type: Functional
+Importance: Critical
+
+Steps:
+  * Validate manual approval gate deployment
+  * Apply
+     | S.NO | resource_dir                                             |
+     |------|----------------------------------------------------------|
+     | 1    | testdata/manualapprovalgate/manual-approval-pipeline.yaml|
+  * Start the manual-approval-pipeline pipeline
+  * Approve the manual-approval-pipeline
+  * Validate the manual-approval-pipeline for "Approved" state
+  * Verify the latest pipelinerun for "successful" state
```

Comment:
@VeereshAradhya why did you introduce this step and didn't use `Verify pipelinerun <table>` in Tutorial tests?

Reply:
We also have `Verify pipelinerun is <status>` in `specs/concepts`, how does this work?

Reply:
The `Verify pipelinerun <table>` step requires the name of the pipelinerun. The step that I added takes the latest pipelinerun and verifies the status